pkg/csv2lp/multi_closer.go

```diff
@@ -0,0 +1,33 @@
+package csv2lp
```

Comment:
Is this file actually used anywhere? If not we should remove...

Reply:
It is not used internally by the library. It is referenced in README.MD (Support Existing CSV files) since it helps compose a single input reader out of to compose multiple Readers and ReadClosers.

[csv2lp.MultiCloser](./multi_closer.go) helps with closing multiple io.Closers (files) on input, because [it is not available OOTB](https://github.com/golang/go/issues/20136)